### ISSUE TITLE
[Windows only] Can only run tests with Unittest or Pytest running tests serially

### ISSUE THREAD

**Issue by Rowlando13** [NREL/mappymatch#44]
To reproduce the the issue run ```pytest tests``` from the root directory. 

The error is ' OSError: [WinError 6] The handle is invalid'

The cause is pytest by default runs tests in parallel. In parallel, the omnx logger is not able to get a handle.

**Comment by Rowlando13** [comment]
Maintainers decided that will just use Unittest features for the time being. Movement to pytest specific features will require discussion. Moved to low priority. Only become high priority if test time becomes burdensome.
